src/channels-api.js

```diff
@@ -457,10 +457,10 @@ class ChannelsApi implements IStreamingAPI {
     return {stream, payload};
   }
 
-  _buildRequestResponseSelector(stream: string, requestId: string) {
+  _buildRequestResponseSelector(stream: string, request_id: string) {
```

Comment:
Again here — use `requestId` as the parameter name, but `payload: {request_id: requestId}` below.